pytest_elk_reporter.py

```diff
@@ -317,7 +317,7 @@ def pytest_internalerror(self, excrepr):
         test_data = dict(
             timestamp=datetime.datetime.utcnow().isoformat(),
             outcome="internal-error",
-            faiure_message=excrepr,
+            faiure_message=str(excrepr),
```

Comment:
Would it print the whole backtrace ? or just the line of the exception?

Reply:
the whole thing. see https://github.com/pytest-dev/pytest/blob/12e061e2e838fb2c17d54df40a4a11aeb8723c6b/src/_pytest/_code/code.py#L1093-L1099